src/planscape/plan/views.py

```diff
@@ -578,6 +579,44 @@ def create_scenario(request: HttpRequest) -> HttpResponse:
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
 
 
+@csrf_exempt
+def update_scenario(request: HttpRequest) -> HttpResponse:
+    try:
+        # Check that the user is logged in.
+        owner = get_user(request)
+
+        body = json.loads(request.body)
+        scenario_id = body.get('id', None)
+        if scenario_id is None or not (isinstance(scenario_id, int)):
+            raise ValueError("Must specify scenario_id as an integer")
+
+        scenario = Scenario.objects.get(id=scenario_id)
+        if scenario.owner != owner:
+            raise ValueError(
+                "You do not have permission to view this project.")
+
+        notes = body.get("notes", None)
+        status = body.get("status", None)
+
+        if notes is not None and not (isinstance(notes, str)):
+            raise ValueError("Notes must be a string value")
+
+        if status is not None and not (isinstance(status, int)):
```

Comment:
is there a way to validate ScenarioStatus beyond whether it's an integer?

Reply:
(actually: I'm guessing ScenarioSerializer will do it - wdyt about adding a testcase for status=99?)

Reply:
added! i thought about too and came to the same conclusion

Reply:
verified that this returns 
Ill-formed request: {\'status\': [ErrorDetail(string=\'"99" is not a valid choice.\', code=\'invalid_choice\')]}'